Graphics/GraphicsEngineOpenGL/include/GLPipelineResourceLayout.hpp

```diff
@@ -328,9 +331,9 @@ class GLPipelineResourceLayout
 /*34*/ OffsetType m_ImageOffset         = 0;
 /*36*/ OffsetType m_StorageBufferOffset = 0;
 /*38*/ OffsetType m_VariableEndOffset   = 0;
-/*40*/ std::array<Int8, 6> m_ProgramIndex = {{-1, -1, -1, -1, -1, -1}};
-/*46*/ Uint8      m_NumPrograms         = 0;
-/*48*/
+/*40*/ std::array<Int8, NUM_SHADER_TYPES> m_ProgramIndex;
```

Comment:
FYI, the size of the struct will be 56 bytes, which is why I have the offset as a hint